src/components/Deployment/Deployment.tsx

```diff
@@ -0,0 +1,262 @@
+        const account = await fetchDeploymentData(chainId);
+        if (account) {
+          result.push({ ...account, chainId, title, iconUrl });
+        }
+      }
+      return result;
+    } catch (err) {
+      return [];
+    }
+  };
+
+  const getDeploymentData = async () => {
+    const data: IDeployChain[] = await getDeploymentStatus();
+    if (!data) return;
+    const [undeployed, deployed] = data.reduce(
+      (acc: IDeployChain[][], curr: IDeployChain) => {
+        if (curr.state ===  AccountStates.UnDeployed) {
+          acc[0].push(curr);
+        } else {
+          acc[1].push(curr);
+        }
+        return acc;
+      },
+      [[], []]
+    );
+    setUndeployedChains(undeployed);
+    setDeployedChains(deployed);
+  };
+
+  const deploy = async () => {
+    if (!selectedChain || !accountAddress) return;
+    
+    const sdk = getSdkForChainId(selectedChain);
+    if (!sdk) {
+      setErrorMessage('Failed to proceed with selected actions!');
+      return;
+    }
+    try {
+      await deployAccount(sdk);
+      await getDeploymentData();
+    } catch (err) {
+      setErrorMessage('Failed to proceed with selected actions!');
+      hideModal();
+    }
+  };
+
+  const handleClose = () => {
+    setErrorMessage('');
+  };
+
+  const hideModal = () => {
+    setConfirmModal(false);
+  };
+  
+  useEffect(() => {
+    getDeploymentData();
+  }, []);
+
+  return (
+    <Card title="Deployments" marginBottom={20}>
+      {errorMessage && <ErrorMessage errorMessage={errorMessage} onClose={handleClose} />}
+      {!!confirmModal && (
+        <Modal>
+          <Paragraph>Deploy your wallet on Ethereum to sign messages</Paragraph>
+          <PrimaryButton
+            background={theme?.color?.background?.selectInputScrollbar}
+            color={theme?.color?.text?.listItemQuickButtonPrimary}
+            display="flex"
+            justifyContent="center"
+            onClick={deploy}
+          >
+            Deploy
+          </PrimaryButton>
+          <br />
+          <SecondaryButton color={theme.color?.text?.card} onClick={hideModal}>
+            Cancel
+          </SecondaryButton>
+        </Modal>
+      )}
+      <Wrapper>
+        <Header>
+          <IoMdCheckmark size={20} style={{ marginRight: '2px' }} />
+          <Label>Deployed</Label>
+        </Header>
+        <Body>
+          {deployedChains.map(({ iconUrl, title }) => {
+            return (
+              <Section>
+                <RoundedImage url={iconUrl} title={title} size={24} />
+                {title}
+                <span style={{ marginLeft: 'auto' }}>Auth Chain</span>
+              </Section>
+            );
+          })}
+        </Body>
+      </Wrapper>
+      <Wrapper>
+        <Header>
+          <CgSandClock size={20} style={{ marginRight: '2px' }} />
+          <Label>Not Deployed</Label>
+        </Header>
+        <Body>
+          {undeployedChains.map(({ chainId, iconUrl, title }) => {
+            return (
```

Comment:
can return just jsx component without `return`
```suggestion
          {undeployedChains.map(({ chainId, iconUrl, title }) => (
```